docs/src/examples/connectivity/index.mdx

```diff
@@ -0,0 +1,127 @@
+---
```

Comment:
Excellent article @MaikuB. Wonderfully written and easy to follow 🥇

Reply:
cheers :)